Week-2/Exercise1/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+    <div id="flex-container">
+        <header>
+            <h2>Pesto</h2>
+            <div>A Program to help you the best version of yourself.</div>
+        </header>
+        <section id="section1">
+            <div class="title"><strong>Join a lifelong community to grow.</strong></div>
```

Comment:
Inside section, you can also use article for semantics

Reply:
Did the change and committed